flask_monitoringdashboard/core/profiler/stacktraceProfiler.py

```diff
@@ -24,12 +24,13 @@ class StacktraceProfiler(threading.Thread):
     This is used when monitoring-level == 2 and monitoring-level == 3
     """
 
-    def __init__(self, thread_to_monitor, endpoint, ip, group_by, outlier_profiler=None):
+    def __init__(self, thread_to_monitor, endpoint, ip, group_by, request_data= {}, outlier_profiler=None):
```

Comment:
Change to `request_data=None`

Then Line 33 becomes: `self._request_data = request_data or {}`

For more info: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments